### PR TITLE
Accept A-api-request-uuid and add it to presigned when available

### PR DESCRIPTION
**Pull request by jeffersonwhite**

### CODE REVIEW

**Comment by jeffersonwhite** [comment]
There's also a corresponding change in TEA:
https://github.com/asfadmin/thin-egress-app/pull/807

File: rain_api_core/egress_util.py [hunk 1]
```diff
@@ -47,6 +56,13 @@ def get_presigned_url(session, bucket_name, object_name, region_name, expire_sec
         "X-Amz-SignedHeaders=host"
     ])
 
+    # Add the api_request_uuid header only if it is available
```
Comment (reweeden) [inline rain_api_core/egress_util.py:59]: I don't think the comment is necessary. The code is straight forward enough that it is clear what is happening without it.

**Review by mattp0 — APPROVED** [review]
I think we should merge rain-api-core then the TEA PR.

**Review by reweeden — APPROVED** [review]

**Comment by jeffersonwhite** [comment]
@mattp0 @reweeden The failure in the code coverage test looks to be unrelated, thoughts on what to do? Can we retrigger the test?

**Review by mattp0 — APPROVED** [review]

**Comment by mattp0** [comment]
> @mattp0 @reweeden The failure in the code coverage test looks to be unrelated, thoughts on what to do? Can we retrigger the test?

Its failing on the last step of the code cov reporter, I am going to merge this and we will take care of the code cov reporting action issue.

**Pull request merged**